SoftwareGuide/Latex/DesignAndFunctionality/Transforms.tex

```diff
@@ -271,7 +267,7 @@ \subsection{Identity Transform}
 
 \begin{table}
 \begin{center}
-\begin{tabular}{\tableconfiguration}
+\begin{tabular}{ | p{3cm} | p{1.8cm} | p{2.5cm} | p{4cm} |}
```

Comment:
Does the error reported in 030ae1a312d48134950e47a598b363046001afa7 depend somehow on the platform/MikTeX configuration? I guess it did work at some point?

I've tested a simple case [here](https://www.overleaf.com/read/pydrpqqwpmhq) and I got no errors.

I think having a definition for it enhances readability. Although it looks like it is only used in this file, if the issue is related to where the definition lies, I'd then define it in some common/style file (e.g. `00-Preamble-Common.tex`).

Reply:
Yes, it was a good idea to avoid duplication, but it just did not compile with TeX Live as opposed to MikTeX, even if the definition was available.